lib/bcoin.js

```diff
@@ -161,71 +165,71 @@ class LedgerBcoin {
   }
 
   /**
-   * start composing tx
+   * Start composing tx
    * @async
-   * @param {Buffer} tx
-   * @param {String} inputKey - Prevout to key
-   * @param {bcoin.Script} prev - prev script for current input
-   * @param {Object} tis - trusted inputs map[prevoutKey] = trustedInput
-   * @param {Boolean?} [isNew=false] - is new transaction
+   * @param {MTX} tx
+   * @param {Object} tis
+   * @param {Boolean} [isNew=false]
+   * @param {Boolean} [witness=false] - is v1 tx
    * @throws {LedgerError}
    */
 
-  async hashTransactionStart(tx, inputKey, prev, tis = {}, isNew = false) {
+  async hashTransactionStart(tx, tis, isNew, hasWitness) {
```

Comment:
Wondering if there's a better name for this method? maybe just `hashTX` or `startTXHash` (also, the naming convention in bcoin tools so far I think is usually `TX` for transaction right?)